src/parsers/power_system_table_data.jl

```diff
@@ -668,8 +699,19 @@ function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames,
     )
 
     capacity = gen.active_power_limits_max
-    startup_cost = gen.startup_heat_cold_cost * fuel_cost * 1000
-    shutdown_cost = 0.0
+    startup_cost = get(gen, :startup_cost, nothing)
+    if isnothing(startup_cost) && hasfield(typeof(gen), :startup_heat_cold_cost)
```

Comment:
This means that startup_heat_cold_cost is a required field.  Another way to do this is to define a default function get_startup_heat_cold_cost.  Then, users could extend in their own way.  We should be consistent on whether fields are required.

Reply:
Agreed. In this effort, I've realized that we have a pretty big mis-alignment on required table columns and required struct fields. I think we need a distinct effort to resolve that